fs/v9fs/Kconfig

```diff
@@ -21,4 +21,9 @@ config V9FS_VIRTIO_9P
 	depends on DRIVERS_VIRTIO
 	default n
 
+config V9FS_SOCKET_9P
```

Comment:
```suggestion
config V9FS_SOCKET
```

Reply:
but viritio is V9FS_VIRTIO_9P

Reply:
The current naming format is `V9FS_[Transport NAME]_9P`
If we change to `V9FS_SOCKET`, then VIRTIO should also be changed to `V9FS_VIRTIO`

Reply:
Ok, let's create a new patch

Reply:
OK, let us keep this style